templates/scenarios/ts/spfx-tab/src/README.md

```diff
@@ -22,18 +22,14 @@ Learn more about the changes in this pre-release at [https://aka.ms/teamsfx-v5.0
 
 The SharePoint Framework (SPFx) is a page and web part model that provides full support for client-side SharePoint development, easy integration with SharePoint data, and extending Microsoft Teams. This project applies SPFx to Teams personal tab and group tab support.
 
-## Used SharePoint Framework Version
-
-![version](https://img.shields.io/badge/version-1.16.1-green.svg)
-
 ## Applies to
 
 - [SharePoint Framework](https://aka.ms/spfx)
 - [Microsoft 365 tenant](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/set-up-your-developer-tenant)
 
 ## Prerequisites
 
-> - [Node.js](https://nodejs.org/en/), supported versions: 16
+> - [SharePoint Framework development environment](https://aka.ms/teamsfx-spfx-dev-environment-setup)
 > - An Microsoft 365 account. Get your own free Microsoft 365 tenant from [Microsoft 365 developer program](https://developer.microsoft.com/en-us/microsoft-365/dev-program)
```

Comment:
Download Visual Studio Code and install [Teams Toolkit for Visual Studio Code Extension](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension).

Reply:
Download [Visual Studio Code](https://code.visualstudio.com) and install [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit) or download TeamsFx CLI. - how about this one?

Reply:
ok